v2/openapi/fb-unified-openapi.yaml

```diff
@@ -2890,12 +2895,35 @@ components:
           $ref: '#/components/schemas/AssetReference'
         transferMethod:
           enum:
-            - InternalTransfer
             - PeerAccountTransfer
-    CrossAccountTransferDestination:
+    InternalTransferCapability:
+      type: object
+      required:
+        - asset
+        - transferMethod
+      properties:
+        asset:
+          $ref: '#/components/schemas/AssetReference'
+        transferMethod:
+          enum:
+            - InternalTransfer
+    PeerAccountTransferDestination:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/PeerAccountTransferCapability'
+        - type: object
+          required:
+            - accountId
+            - amount
+          properties:
+            accountId:
+              type: string
+            amount:
+              $ref: '#/components/schemas/PositiveAmount'
```

Comment:
Split it into `PeerAccountTransferAddress` that contains only the account ID and `PeerAccountTransferDestination` that inherits `PeerAccountTransferAddress` and adds to it the amount property. This is needed both for the uniformity and for deposit support, that we'll add separately.

Reply:
Done